### PR TITLE
bug #137: Fix complaint form

### LINKED ISSUE #137: [BUG] Fix complaint form
**Preconditions:**

**Steps to reproduce**
1. Go to Teacher page
2. Click on Leave a complaint

**Actual result:**
image
image

**Expected result:**
- designed dropdown options
- labels and description for fields

### PR DESCRIPTION
**Pull request by sifes**
- designed dropdown options
- labels and description for fields

closes #137

### CODE REVIEW

**Review by hoshion — APPROVED** [review]

File: fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx [hunk 1]
```diff
@@ -31,68 +32,68 @@ const ComplaintPopup: FC<ComplaintPopupProps> = ({
...
+        await TeacherAPI.postTeacherComplaint(
+          teacherId,
+          convertEmptyStringToUndefined(data),
+        );
+        toast.success(
+          'Скарга надіслана',
+          'Ми успішно отримали скаргу на викладача та вже працюємо над нею',
+          5000,
+        );
+        setIsOpen(false);
```
Comment (kujo205) [inline fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx:49]: It's not clear what is open, come up with a better name

File: fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx [hunk 2]
```diff
@@ -31,68 +32,68 @@ const ComplaintPopup: FC<ComplaintPopupProps> = ({
...
-        'Скарга надіслана',
-        'Ми успішно отримали скаргу на викладача та вже працюємо над нею',
-        5000,
-      );
-      setIsOpen(false);
-    } catch (error) {
-      displayError(error);
-    }
-  };
+  const handleSubmit = useCallback(
```
Comment (kujo205) [inline fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx:37]: Why do you need to use useCallback here, this component is very rarely updated

File: fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx [hunk 3]
```diff
@@ -31,68 +32,68 @@ const ComplaintPopup: FC<ComplaintPopupProps> = ({
...
+            }}
             secondButton={
-              <Box>
+              <Box sx={{ marginTop: '18px', marginBottom: '30px' }}>
                 <Button
-                  size={ButtonSize.MEDIUM}
+                  size={isTablet ? ButtonSize.SMALL : ButtonSize.MEDIUM}
                   variant={ButtonVariant.FILLED}
                   text="Лишити скаргу"
-                  type="submit"
```
Comment (kujo205) [inline fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx:95]: Is it now working with type="submit"? Strange

**Review by kujo205 — CHANGES_REQUESTED** [review]
Good overall

File: fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx (hunk 3 above)
Comment (sifes) [reply fictadvisor-web/src/components/pages/personal-teacher-page/personal-teacher-tabs/components/complaint-popup/ComplaintPopup.tsx:95]: it doesnt work with type="submit"

**Review by kujo205 — APPROVED** [review]
Approved

**Review by hoshion — APPROVED** [review]

**Pull request merged**